cognite/neat/core/_client/_api/statistics.py

```diff
@@ -0,0 +1,91 @@
+import itertools
+from typing import overload
+
+from cognite.client._api_client import APIClient
+from cognite.client._cognite_client import CogniteClient
+from cognite.client.config import ClientConfig
+from cognite.client.data_classes.data_modeling.ids import _load_space_identifier
+from cognite.client.utils.useful_types import SequenceNotStr
+
+from cognite.neat.core._client.data_classes.statistics import (
+    ProjectStatsAndLimits,
+    SpaceInstanceCounts,
+    SpaceInstanceCountsList,
+)
+
+
+class StatisticsAPI(APIClient):
```

Comment:
nitpick (not blocker): For other people reviewing this code, the name of the class might give the impression that this is an statiticsAPI inside neat, but this is actually just consuming from `/models/statistics` endpoint from data modelling. Maybe some comments or even changing the name of the class might help